be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -534,6 +535,7 @@ Status ExchangeSinkOperator::set_finishing(RuntimeState* state) {
 
 void ExchangeSinkOperator::close(RuntimeState* state) {
     COUNTER_SET(_network_timer, _buffer->network_time());
+    COUNTER_SET(_exchange_timer, _buffer->lifecycle_time());
```

Comment:
The name of `_exchange_timer` and `lifecycle_time` are very confusing me.

Reply:
`exchange_timer` are similar to `ScanTime` of `OLAP_SCAN`
`lifecycle_time` means the entire time from entering the SinkBuffer queue to receiving the response.

Do you have a better suggestion for naming these two?

Reply:
So the meaning of `lifecycle_time` is same with the `wait response time` in old  engine?

Reply:
Roughly the same with a small difference:
* `old engine`: `wait response time` only
* `pipeline engine`: `wait in queue time` +  `wait response time`

Reply:
for user understand easily, I think   wait response time or wait time would be better.  lifecycle is a little abstract.

Reply:
Ok, I'll update